src/test/resources/TestNG.xml

```diff
@@ -7,6 +7,7 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="Homework18"/>
```

Comment:
//comment from hw team:  Homework requirement #18: XML File Configurations requires you to `Use the <methods> & <include> tags` in the XML file, but currently I have not seen you use it.

TestNg provides an option to include or exclude Groups, Test Methods, Classes and Packages using include and exclude tags by defining in testng.xml.
You will create a Class with Test Methods. and include test methods into xml file. in your Homewok18 you already have the playSong method, so you need to include that method into xml file

//HINT: 

```
<class name="Homework18">
                <methods>
                    <include name="playSong"/>
                </methods>
            </class>
```

Please check and update it. 
💖Best wishes from the HW team (Khanh, khanh@testpro.io)!💖